src/objects/zcl_abapgit_object_intf.clas.abap

```diff
@@ -244,12 +251,7 @@ CLASS zcl_abapgit_object_intf IMPLEMENTATION.
       iv_object_name = lv_object
       iv_language    = mv_language ).
     IF lines( lt_lines ) > 0.
```

Comment:
What about the IF here? Can be removed, right? @larshp

Reply:
yes